src/goSuggest.ts

```diff
@@ -55,7 +55,22 @@ export class GoCompletionItemProvider implements vscode.CompletionItemProvider {
 				// prevent completion when typing in a line comment
 				const commentIndex = lineText.indexOf('//');
 				if (commentIndex >= 0 && position.character > commentIndex) {
-					return resolve([]);
+					if (!lineText.trim().startsWith('//')) {
```

Comment:
To avoid a lot of nesting, you can pull your code out and place it before line 55. You can start with
```
// triggering completions in comments on exported members
if (lineText.trim().startsWith('//') && (position.line + 1 < document.lineCount)) {
   let nextLine = .....
}
```